eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheetSlot.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { SlotInterface } from "eisbuk-shared";
+import { CustomerWithAttendance } from "@/types/components";
+
+import { getSlotTimespan } from "@/utils/helpers";
+import Box from "@material-ui/core/Box";
+import ProjectIcon from "@/components/global/ProjectIcons";
+import Typography from "@material-ui/core/Typography";
+
+import { slotsLabels } from "@/config/appConfig";
+import {
+  slotTypeLabel,
+} from "@/lib/labels";
+/** @TODO remove this and import same type when it's merged*/
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+
+const AttendanceSheetSlot: React.FC<Props> = ({
+  customers,
+  intervals,
+  type,
+  notes,
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const slotLabel = slotsLabels.types[type];
+
+  const timeString = getSlotTimespan(intervals);
+
+  /** UI todos:
+   * @TODO in page component render alternative background shades for slot table
+   * @TODO center each slot timestring (different width percentages for table cells doesn't work)
+   */
+  return (
+    <Table aria-label="simple table">
+      <TableHead>
+        <TableRow className={classes.slotHeading}>
+          <TableCell className={classes.bold}>
+            <span className={classes.margins}>{timeString} </span>
+          </TableCell>
+          <TableCell>
+            <Box
+              className={[classes.flexCenter, classes.typeLabel].join(" ")}
+              flexGrow={1}
+              pl={1}
+              pr={1}
+            >
+              <ProjectIcon
+                className={classes.typeIcon}
+                icon={slotLabel.icon}
+                fontSize="small"
+              />
+              <Typography
+                className={classes.type}
+                key="type"
+                color={slotLabel.color}
+              >
+                {t(slotTypeLabel[type])}
+              </Typography>
+            </Box>
+            {notes && <span className={classes.flexCenter}> ({notes})</span>}
+          </TableCell>
+        </TableRow>
```

Comment:
I like the solution to show one time string on one end, and the rest on the other, but you can achieve that with way less markup. Have them all in one cell (the whole row), for that cell, you can use the `justify-content: "space-between"` and then have one `<Typography>` for timestring and one `div` containing type and notes.

This will also solve the problem of alignment: timestring is a bit indented with respect to interval strings.